libs/gi/ui/src/components/artifact/ArtifactCard.tsx

```diff
@@ -347,6 +348,17 @@ export function ArtifactCardObj({
                   />
                 )
             )}
+            <Typography
+              variant="caption"
+              sx={{ display: 'flex', gap: 1, alignItems: 'center' }}
+            >
+              <ColorText color="secondary" sx={{ flexGrow: 1 }}>
+                Total Rolls
+              </ColorText>
+              <SqBadge color={totalRolls !== 0 ? 'success' : 'secondary'}>
+                {totalRolls}
```

Comment:
If its undefined, maybe make it say {guessedTotalRolls} (ambiguous) or something